src/useObserver.ts

```diff
@@ -7,24 +7,24 @@ import { useForceUpdate, useUnmount } from "./utils"
 export type ForceUpdateHook = () => () => void
 
 export interface IUseObserverOptions {
-    customForceUpdateHook?: ForceUpdateHook
+    useForceUpdate?: ForceUpdateHook
 }
 
 export function useObserver<T>(
     fn: () => T,
-    baseComponentName?: string,
+    baseComponentName: string = "observed",
     options?: IUseObserverOptions
```

Comment:
If you do it like this, the default will be applied from the import and it's still possible to override.
```ts
options: IUseObserverOptions = {
  useForceUpdate
}
```

Reply:
same as the other comment:

not quite, if that's done and a new property is ever added to options and options does not contain useForceUpdate then that would break

e..g
```
useObserver(..., {
  someOtherOption: ...,
  // no useForceUpdate since it is optional
}

const forceUpdate = options.useForceUpdate() // useForceUpdate will be undefined 
```

Reply:
Correction...This way it either takes one specified by the user or imported one if undefined.
```
options: IUseObserverOptions = {
  useForceUpdate = useForceUpdate
}
```

Reply:
That doesn't compile :(

Reply:
Ah nevermind, just realized what you mean. Let's do this at least so there is always an object for options.

```
export function useObserver<T>(
    fn: () => T,
    baseComponentName: string = "observed",
    options: IUseObserverOptions = {}
): T {
    if (isUsingStaticRendering()) {
        return fn()
    }

    const wantedForceUpdateHook = options.useForceUpdate || useForceUpdate
```

Reply:
done, but used an EMPTY_OBJECT constant so it won't waste time creating an empty object when there are no options (which is the usual)